packages/phone-number-privacy/signer/README.md

```diff
@@ -49,20 +49,36 @@ Use the following to configure the Google Secret Manager. To authenticate with G
 
 ### Setup
 
-The service requires a connection to a SQL database. The SQL connection parameters should be configured with the `DB_*` configs stated above.
+The service requires a connection to a secret store and to a SQL database. The SQL connection parameters should be configured with the `DB_*` configs stated above.
 
-Once the database connection is established, the tables can be set up with the migration script. Set the DB env variables and then run: `yarn db:migrate`
+#### Running locally or without docker
 
-### Running locally
+To run without docker, or for development, start by git cloning the celo-monorepo. Next, run `yarn` from the monorepo root to install dependencies.
 
-- `yarn install`
-- `yarn start`
+Before the service can work, the db migrations must be run once. Set the DB env variables and then run: `yarn db:migrate`
```

Comment:
this will need to happen whether running with or without docker, so perhaps move above this section.  or are you planning to update the docker instructions to include the db setup as well?

Reply:
The reason I put it here is that the instructions for running them differ for docker / non-docker. Maybe I can reword a bit to make it more clear